src/pages/ClientProfile.js

```diff
@@ -123,11 +136,11 @@ class ClientProfile extends React.Component{
   };
 
   render(){
-    const { data } = this.state
+    const { profileData } = this.state
 
     return(
       <>
-        {!!data && data.map(({ _id, name, lastName, email }) => {
+        {!!profileData && profileData.map(({ id, clientName, lastName, email, address, phone, identification, birthday, payType })=> {
```

Comment:
profileData debe ser un arreglo o un objeto? OJO con este cambio, ya lo hemos hablado varias veces y he dejado varios comentarios sobre la misma línea en muchos PRs

Reply:
Simon es que esto no va a cambiar en mi PullRequest, esto es algo que esta trabajando Sergio, pero como yo cree una rama a partir de estas modificaciones se trajo también estos errores.

Tu me dirás si desde cero creo una rama de develop y hago mis cambios que son Sign-up y Sign-in

Reply:
Es lo ideal, porque acá estamos haciendo merge de cambios que dependen del código que todavía no está en develop